### ISSUE TITLE
Ark: Debugger: Ensure the control is back to console before disconnect

### ISSUE THREAD

**Issue by yutannihilation** [posit-dev/positron#4010]
Ark's ability to debug into C/C++/Rust code is amazing!

This might be just because I'm not familiar with the debugger, but my R process hangs when I disconnect during the debugger's step is still inside a Rust function. Is it possible to abort (or continue to the end?) before disconnect?

https://github.com/user-attachments/assets/27320ae4-2d79-4292-abf1-9d5f2c1a109d

![image](https://github.com/user-attachments/assets/986eb31e-9cde-44e6-875d-d79702871745)

**Comment by lionel-** [comment]
hmm this is not really in our control. I think if you click on Continue rather than Disconnect it should work?

**Comment by yutannihilation** [comment]
Yes, it works. So, I don't mind this because I learned how to deal with this, but I'm afraid this is a bit tricky to new users.

**Comment by lionel-** [comment]
Great. I think we're unlikely to be making changes there but I made a note to at least mention this in our documentation for C/C++ development in packages: https://github.com/posit-dev/positron/issues/4015

**Comment by yutannihilation** [comment]
I see, thanks!
